lib/git-hooks.js

```diff
@@ -3,6 +3,12 @@ var util = require('util');
 var spawn = require('child_process').spawn;
 var fs = require('fs');
 var fsHelpers = require('./fs-helpers');
+var execSync = require('child_process').execSync;
+if (!execSync) {
+    execSync = require('runsync').exec;
```

Comment:
I really don't like this `preinstall` script. Let's implement it without an external module. Async version is fine.